chains/evm/contracts/pools/TokenPool.sol

```diff
@@ -1149,17 +1150,33 @@ abstract contract TokenPool is IPoolV2, Ownable2StepMsgSender {
     );
   }
 
-  /// @notice Withdraws all accumulated pool fees to the specified recipient.
-  /// @dev For burn/mint pools, this transfers the entire token balance of the pool contract.
-  /// lock/release pools should override this function with their own accounting mechanism.
-  /// @param recipient The address to receive the withdrawn fees.
-  function withdrawFees(
-    address recipient
-  ) external virtual onlyOwner {
-    uint256 amount = getAccumulatedFees();
-    if (amount > 0) {
-      getToken().safeTransfer(recipient, amount);
-      emit PoolFeeWithdrawn(recipient, amount);
+  /// @notice Withdraws accrued pool token fees and outstanding balances of the provided fee tokens.
+  /// @dev Restricted to the owner to align with operational controls around fee management. For burn/mint pools, the
+  /// accumulated pool token fees correspond to this contract's balance and are sent to the supplied `recipient`.
+  /// lock/release pools should override this function with custom accounting for pool token fees.
```

Comment:
LR pools will have lockboxes, right? So no pool will ever hold funds again, meaning this comment isn't correct

Reply:
Also the code below can be simplified if we assume everything will use lockboxes or mint/burn.